src/main/java/stashpullrequestbuilder/stashpullrequestbuilder/StashRepository.java

```diff
@@ -243,8 +243,13 @@ private Boolean isPullRequestMergable(StashPullRequestResponseValue pullRequest)
       StashPullRequestMergableResponse mergable =
           client.getPullRequestMergeStatus(pullRequest.getId());
       boolean res = true;
-      if (trigger.isCheckMergeable()) res = res && mergable.getCanMerge();
-      if (trigger.isCheckNotConflicted()) res = res && !mergable.getConflicted();
+      if (trigger.isCheckMergeable()) {
+        res = res && mergable.getCanMerge();
```

Comment:
minor: maybe change to `&=` while we'r at it?

Reply:
Done. Also rebased. I like rebasing on top of my merged commits :smiley:

Reply:
I can only hope this happens more often :)